lib/aiko/led.py

```diff
@@ -130,6 +136,9 @@ def pixel_get(x=0):
     x = length_x * (x // length_x + 1) - (x - length_x * (x // length_x)) - 1
   return np[x]
 
+def length_get():
+    return length
+
```

Comment:
`aiko.led.length` works… sure, it's not at all encapsulated, but bytes are scarce, no?

Reply:
yes, this was also to try namespace stuff, but this one is entirely useless, I removed it in my version.